trimmer_decorator.rb

```diff
@@ -0,0 +1,11 @@
+require_relative_path 'decorator'
```

Comment:
```suggestion
require_relative 'decorator'
```

- [ ] Kindly change the path to `require_relative 'decorator'`, you should use require_relative for local files 👍🏿